site/utils/index.ts

```diff
@@ -225,3 +226,15 @@ export const removeDuplicates = <T, K extends keyof T>(arrayToRemoveDuplicates:
     arrayToRemoveDuplicates.filter(
         (value, index, self) => index === self.findIndex((item) => item[key] === value[key]),
     );
+
+export const filterVehicleModes = (showUnderground?: boolean) =>
+    VEHICLE_MODES.filter((v) => (showUnderground ? true : v.value !== VehicleMode.underground));
+
+export const filterStopList = (stops: Stop[], vehicleMode: VehicleMode | Modes, showUnderground?: boolean) =>
+    stops.filter((stop) =>
+        showUnderground && vehicleMode === VehicleMode.underground
+            ? stop.commonName.toLowerCase().includes("underground")
+            : showUnderground && vehicleMode === VehicleMode.tram
+            ? stop.commonName.toLowerCase().includes("tram")
+            : true,
+    );
```

Comment:
```suggestion
export const filterStopList = (stops: Stop[], vehicleMode: VehicleMode | Modes, showUnderground?: boolean) =>
    stops.filter((stop) => {
        if (showUnderground && vehicleMode === VehicleMode.underground) {
            return stop.commonName.toLowerCase().includes("underground");
        }
        if (showUnderground && vehicleMode === VehicleMode.tram) {
            return stop.commonName.toLowerCase().includes("tram");
        } else return true;
    });
```

I think in this case if statements are a bit more readable than a nested ternary